src/js/tech/html5.js

```diff
@@ -469,7 +469,26 @@ class Html5 extends Tech {
    * @return {Number}
    * @method duration
    */
-  duration() { return this.el_.duration || 0; }
+  duration() {
+    // Android Chrome will report duration as Infinity for VOD HLS until after
+    // playback has started, which triggers the live display erroneously.
+    // Return NaN if playback has not started and trigger a durationupdate once
+    // the duration can be reliably known.
+    if (this.el_.duration === Infinity &&
+      browser.IS_ANDROID && browser.IS_CHROME) {
+      if (this.el_.currentTime === 0) {
+        let checkProgress = () => {
```

Comment:
The newly released version of videojs-standard will complain about the use of `let` for names that are never reassigned (`const` is preferred).